rapier-compat/build-rust.sh

```diff
@@ -41,4 +41,10 @@ fi
 
 # Working dir in wasm-pack is the project root so we need that "../../"
 
-wasm-pack build --target web --out-dir "../../rapier-compat/builds/${dimension}d${feature_postfix}/wasm-build" "$rust_source_directory"
+if [[ $feature == "simd" ]]; then
+    export additional_rustflags='-C target-feature=+simd128'
```

Comment:
I tried `export RUSTFLAGS=[...]` but this didn't work, I think my shell expertise is needing an update, there's probably a slightly cleaner way to achieve this.

in the meantime just removing the `export` in the only self-improvement I can come up with.